bokehjs/src/coffee/renderer/glyph/glyph_renderer.coffee

```diff
@@ -59,7 +59,7 @@ class GlyphRendererView extends PlotWidget
         @request_render()
     )
 
-  have_selection_glyphs: () -> true
+  have_selection_glyphs: () -> @selection_glyph? && @nonselection_glyph?
```

Comment:
@bryevdv  This, and the extra call to `@have_selection_glyphs()` below does the trick for allowing these glyphs to be null.